manifests/server/puppetserver.pp

```diff
@@ -145,7 +145,7 @@
   Optional[Integer[1]] $max_open_files = $puppet::server::max_open_files,
   Optional[Stdlib::Absolutepath] $versioned_code_id = $puppet::server::versioned_code_id,
   Optional[Stdlib::Absolutepath] $versioned_code_content = $puppet::server::versioned_code_content,
-  Boolean $disable_fips = $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '8',
+  Boolean $disable_fips = $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] != '7',
```

Comment:
Given we don't support EL7 anymore, should this just be `true`?

Reply:
In this module we still support EL7 for puppetserver. Considering the options I think the current implementation is good enough. You could use `versioncmp` to test for `>=`, but I think this is close enough now.

Reply:
Oh right, this is puppet, not candlepin 🙈